lib/cisco_node_utils/cmd_ref/radius_server.yaml

```diff
@@ -1,66 +1,101 @@
 # radius_server.yaml
 ---
-_exclude: [ios_xr]
+_template:
+  ios_xr:
+    get_command: "show running-config radius-server"
+    multiple: true
 
 accounting:
+  _exclude: [ios_xr]
```

Comment:
I don't think so. I don't believe the accounting and authentication bools make sense at all in the context of XR.